### ISSUE TITLE
The distance between cursor and the last code is getting larger and large while typing

### ISSUE THREAD

**Issue by delta4d** [joshnuss/mruby-web-irb#6]
![](http://i.imgur.com/1tkkEqs.png)

I am using Chrome 39.0.2171.95 m

**Comment by joshnuss** [comment]
Works fine on Chrome 40.0.2214.45 beta

What OS? Is it on a fresh screen? or only after running specific command?

**Comment by delta4d** [comment]
OS is windows 7 SP1 ultimate
yes, it is on a fresh screen
the distance is getting larger while typing whatever command

**Comment by joshnuss** [comment]
Hmm, is the window zoomed in at all? I'm not seeing it

**Comment by delta4d** [comment]
the window is maximized

**Comment by joshnuss** [comment]
maybe try to poke around in Web Inspector and see what is causing the padding? (Dont have a windows machine to test with)

**Comment by delta4d** [comment]
I record a gif

![](http://i.imgur.com/OFGLS2f.gif)

is it common problem on windows machine?

**Comment by joshnuss** [comment]
Havent seen it before (though I am on linux)

Can you try using the web inspector (right click the extra spacing area between the cursor and last letting and click "Inspect element" from the context menu)

**Comment by delta4d** [comment]
it shows this

```
<div class="ace_print_margin_layer">
   <div class="ace_print_margin" style="left: 934px; visibility: hidden;"></div>
</div>
```

**Comment by joshnuss** [comment]
Just tested it with http://www.browserstack.com/ on Windows 7/Chrome 39 and it works fine.

Do you have any extensions installed that could be interfering somehow? (You can run without extensions by opening in Incognito Mode CTRL+SHIFT+N)

**Comment by delta4d** [comment]
no, it remains the same, and when I try remove the space and then delete

![](http://i.imgur.com/sbsxLP6.gif)

**Comment by joshnuss** [comment]
Could it be something with locale? Like based on the language your PC is using?

**Comment by joshnuss** [comment]
Do you get the same result on browserstack?

**Comment by delta4d** [comment]
I am running English version of windows, my local is Chinese.(is that gonna be a problem?)
I have not tried browserstack yet.

**Comment by joshnuss** [comment]
Chinese should work fine, but I've never tested it. Maybe try switching to en-us to see if theres any difference.

**Comment by delta4d** [comment]
I may try it later some time, my laptop is running some program to generate data, it could not restart now.
Thanks for answering, I will keep you informed when I test it.

**Comment by delta4d** [comment]
already changed locale to English(US), still the same result.
